processor/src/operations/u32_ops.rs

```diff
@@ -27,18 +27,18 @@ where
 
     /// Pops top two element off the stack, splits both into low and high 32-bit values, checks if both
     /// high are equal to 0, if it passes, put both of them onto the stack, else throws an execution error
```

Comment:
Not a part of this PR, but I'd update this as follows:

Pops top two element off the stack, splits them into low and high 32-bit values, checks if the high  values are equal to 0; if they are, puts the original elements back onto the stack; if they are not, returns an error.